formula-android/src/main/java/com/instacart/formula/fragment/FragmentFlowState.kt

```diff
@@ -11,10 +12,10 @@ import com.instacart.formula.integration.KeyState
  * @param states Last emitted state of each active [FragmentContract].
  */
 data class FragmentFlowState(
-    val activeKeys: List<FragmentKey> = emptyList(),
-    val visibleKeys: List<FragmentKey> = emptyList(),
-    val features: Map<FragmentKey, FeatureEvent> = emptyMap(),
-    val states: Map<FragmentKey, KeyState> = emptyMap()
+    val activeKeys: List<ActiveFragment> = emptyList(),
```

Comment:
The riskiest part of this change is changing from `FragmentKey` type to `ActiveFragment` type (need to find a better name for this).